_config.yml

```diff
@@ -3,7 +3,7 @@ markdown:         kramdown
 highlighter:      rouge
 
 # Setup
-title:            Chiptune Café
+title:            The Chiptune Café
```

Comment:
I removed this for a reason :)
"The" ends up on a line by itself which looks ugly

(case in point with mobile layout)
![image](https://user-images.githubusercontent.com/8082305/70952357-9abd4800-202b-11ea-8ecb-2d979a13e582.png)

Reply:
ok